.github/workflows/release.yaml

```diff
@@ -1,30 +1,25 @@
+---
 name: "Release"
-on:
+on:  # yamllint disable-line rule:truthy
   push:
     tags:
-    - '*'
+      - "v*"
 jobs:
   docker:
     name: "Publish Container Image"
     runs-on: "ubuntu-latest"
     steps:
-    - uses: "actions/checkout@v2"
-    - uses: "docker/setup-qemu-action@v1"
-    - uses: "docker/setup-buildx-action@v1"
-      with:
-        driver-opts: "image=moby/buildkit:master"
-    - uses: "docker/login-action@v1"
-      with:
-        registry: "quay.io"
-        username: ${{ secrets.QUAYIO_USER }}
-        password: ${{ secrets.QUAYIO_PASSWORD }}
-    - uses: "battila7/get-version-action@v2"
-      id: "get_version"
-    - uses: "docker/build-push-action@v2"
-      with:
-        push: true
-        tags: |
-          quay.io/authzed/prom-authzed-proxy:latest
-          quay.io/authzed/prom-authzed-proxy:${{ steps.get_version.outputs.version }}
-          ghcr.io/authzed/prom-authzed-proxy:latest
-          ghcr.io/authzed/prom-authzed-proxy:${{ steps.get_version.outputs.version }}
+      - uses: "actions/checkout@v3"
```

Comment:
should we switch to goreleaser?

Reply:
haha that's the only open issue on this repo: https://github.com/authzed/prom-authzed-proxy/issues/21